docs/azure.md

```diff
@@ -0,0 +1,43 @@
+### Azure
+
+### Export your credentials as environment variables
+
+[Create Azure Service Principal](https://www.terraform.io/docs/providers/azurerm/authenticating_via_service_principal.html) then export the credentials.
+
+```bash
+$ export AZURE_CLIENT_ID=YOUR_AZURE_CLIENT_ID
+$ export AZURE_CLIENT_SECRET=YOUR_AZURE_CLIENT_SECRET
+$ export AZURE_SERVICE_PRINCIPAL=YOUR_AZURE_SERVICE_PRINCIPAL
+$ export AZURE_SUBSCRIPTION_ID=YOUR_AZURE_SUBSCRIPTION_ID
+$ export AZURE_TENANT_ID=YOUR_AZURE_TENANT_ID
+```
+
+## Create the infrastructure
+
+### Initialize the packer environment
+
+```bash
+$ make terraform-init provider=azure env=images-builder
+```
+
+### Create the packer environment
+
+```bash
+$ make terraform-apply provider=azure env=images-builder
```

Comment:
Same thing as before. it seems like the tool details leak and the target is not informing me of what is actually being done (creating a packer env). The core concept is, could the make targets be written on a way that even without docs I would understand what is happening ? Not that this way is impossible to understand, but from the perspective of someone that does not know the project very well it seems a little hard to understand.

Another thing to think is the leverage that another layer like makes gives you. You are already abstracting how things are actually being built, I just want to build them. It would be like compiling C code with make build-gcc, usually the compiler is abstracted away in favor of what you are trying to build.